airbyte-server/src/main/java/io/airbyte/server/ServerFactory.java

```diff
@@ -14,7 +14,12 @@
 import io.airbyte.config.persistence.StatePersistence;
 import io.airbyte.db.Database;
 import io.airbyte.persistence.job.JobPersistence;
+import io.airbyte.server.apis.AttemptApiImpl;
```

Comment:
We should discuss naming conventions on these classes, we (outside of this pr) seem to have a mix of `DefaultInterface` and `InterfaceImpl`.  Personally I lean towards the `Default` prefix, but more I want consistency.